src/relay/transforms/fuse_ops.cc

```diff
@@ -83,6 +83,8 @@ constexpr uint32_t kMaxFusedOps = 256;
 
 static const Op& stop_fusion_op = Op::Get("annotation.stop_fusion");
 
+TVM_REGISTER_PASS_CONFIG_OPTION("relay.max_fuse_depth", Integer);
```

Comment:
consider use the naming convention: `relay.PassName.option_name`

`relay.FuseOps.max_depth`

Reply:
done